examples/fine-tuned-RAG/ModelFinetune.ipynb

```diff
@@ -36,7 +36,7 @@
   },
```

Comment:
<p>This payload construction from dataframe should have been a function — which'd automatically extract a question, embed it and then do this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/openai-cookbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
This is done. Take a look?